--- airstrike-pre6a-src/airstrikerc~	2003-01-18 20:48:11.000000000 +0100
+++ airstrike-pre6a-src/airstrikerc	2005-08-30 20:49:54.000000000 +0200
@@ -4,7 +4,7 @@
 
 
 # The path where the program looks for its data files
-datapath "data"
+datapath "/usr/share/games/airstrike/data"
 
 #window size or fullscreen resolution
 screen.width 800
@@ -26,7 +26,7 @@ nr_players 1
 
 
 #set sound to 1 to enable sound, 0 to disable.
-sound 0
+sound 1
 
 #Level parameters
 level.gravity 0.2
